kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -415,7 +415,9 @@ define(['common/dialogMessages', 'common/jobs', 'common/jobCommChannel'], (
                 }
 
                 // request job updates for the new job
-                self.addListener('job-status', [retry.jobState.job_id]);
+                ['status', 'error', 'does-not-exist'].forEach((type) => {
```

Comment:
add the `error` listener for every job, along with the `status` and `does-not-exist` listeners